railsconf/2020/let_juniors_fail.md

```diff
@@ -0,0 +1,36 @@
+
+Outline:
+
+- Introduction
+  - My journey from junior to where I am now after only 2 years of experience of Rails
+  - Brief story about when my mentor, Nate Hopkins, gave me space to experiment and what an impact that had on my imposter syndrome and confidence
+- Explain what kind of space juniors need to grow
+  - Why failure is important
+  - Fostering mutual respect and creating a safe space
+  - Encouraging experimentation
+- Benefits of creating an environment that encourages creativity
+  - Mental health and burnout
+  - Turning failures into teaching moments
+  - Walking beside, instead of in front of, juniors
+- Conclusion
+
+## Pitch
+
+The more experience you have, the further removed you are from the needs of your junior developers. With only two years experience, I have a fresh perspective on the positive and negative mentorship tactics that many juniors are currently being exposed to. The main premise behind this talk comes from a discussion I had with my team lead about a pattern I had introduced that others didn’t seem to like. The takeaway from that conversation is my lead wanted to give me the space to explore my ideas. If I succeeded, we would have a great new pattern to use moving forward. If I failed, it wasn’t something we couldn’t recover from, and I would have learned a great deal. The key to these two possible outcomes is that regardless of whether I succeeded or failed in my attempt, my mentor was excited to help me realize the lessons I learned from the experiment that would help me to become a better developer moving forward.
+
+The other reason I feel this talk should be considered is because juniors and mids need to see someone like them on the stage. I have been to two RailsConf’s and one RubyConf and took a lot out of the many talks I attended but none of them look like me. Most of the speakers are older, well respected Rubyists and it’s hard to justify taking the time to submit a talk when you think only talks from these more experienced developers will be accepted. By giving this talk, my hope is that I inspire other developers with similar experience to submit their conference talk ideas and continue growing the Ruby community.
+
+### Speaker Information
+
+#### Bio
+
+My name is Andrew Mason, and I am junior Ruby on Rails developer. I am a recent graduate and live and work in Wilmington, NC. I have struggled with perfectionism for my entire life, and I would like to talk about this topic because perfectionism is rarely talked about as being a problem, and more often is portrayed as a positive trait. My hope is that this talk will explain ways to cope with perfectionism, shed some light on the fact that this is an issue, and let others know they are not alone.
```

Comment:
I am a junior Rails developer.

I think it's interesting that you call attention to the notion of perfectionism, but as a heads-up, your bio mentions perfectionism like 5 times while it never occurs in the rest of the document. I would scale back talk of perfection and replace it with mentions of your other hobbies and projects you've worked on, like StimulusReflex and the GitHub actions stuff. I also find it awesome when people finish with something witty. eg. I'm looking to meet people who laugh at my jokes and love long walks on the beach.

Reply:
Ah this is actually an old bio, thanks for pointing this out.

Reply:
The CFP I submitted in 2019 was about perfectionism